lessons/Early-Math/translating/langs/en-us/pages/three-sets-cards.adoc

```diff
@@ -0,0 +1,25 @@
+= Translating Card Sort
+
+++++
+<style>
+table {grid-auto-rows: 1fr;}
+</style>
+++++
+
+To prepare for this activity, give each pair of students an envelope containing three sets of cards. Keep each set of cards (Expressions, Circles of Evaluation, and Words) together with rubber bands or paperclips.
+
+[.FillVerticalSpace, cols="^.^1a,^.^1a,^.^1a", stripes="none", options="header"]
+|===
+|  Expressions		| Circles of Evaluation			| Words
+| @math{15+3}		| @show{(coe '(+ 15 3))}		| the sum of 15 and 3
+| @math{15 / 3}		| @show{(coe '(/ 15 3))}		| the quotient of 15 and 3
```

Comment:
While 15 / 3 looks more like what we see on the circle, in my experience, the students who will be engaging with this material will be far more comfortable with @math{15 \div 3} and, given the simplicity of the expressions being matched to circles it will be an opportunity for them to discover the connection between the symbols.